src/main/java/teammates/common/datatransfer/FeedbackMsqQuestionDetails.java

```diff
@@ -350,20 +408,47 @@ public String getQuestionResultStatisticsHtml(List<FeedbackResponseAttributes> r
             answerFrequency.put(option, 0);
         }
         
+        if (otherEnabled) {
+            answerFrequency.put("Other", 0);
+        }
+        
         int numChoicesSelected = 0;
         for(FeedbackResponseAttributes response : responses){
-            List<String> answerStrings = ((FeedbackMsqResponseDetails)response.getResponseDetails()).getAnswerStrings();
+            List<String> answerStrings = ((FeedbackMsqResponseDetails)response.getResponseDetails()).getAnswerStrings(); 
+            Boolean isOtherOptionAnswer = ((FeedbackMsqResponseDetails) (response.getResponseDetails())).isOtherOptionAnswer();
+            String otherAnswer = "";
+            
+            if (isOtherOptionAnswer) {
+                if (!answerFrequency.containsKey("Other")) {
+                    answerFrequency.put("Other", 1);
+                } else {
+                    answerFrequency.put("Other", answerFrequency.get("Other")+1);
```

Comment:
Remember to add spaces between the `+`, similarly for the one on line 445, 508 and 526 :)